tests/model_monitoring/test_tdengine.py

```diff
@@ -186,35 +241,116 @@ def test_get_records_query(
         start: str,
         end: str,
```

Comment:
```suggestion
        start: datetime.datetime,,
        end: datetime.datetime,,
```
similar change in L397 and L398